### DISCUSSION TITLE
GenericRelation field not recognized as model field

### DISCUSSION THREAD

**Discussion by MatejMijoski** [carltongibson/django-filter#1413] [Q&A]
I have a model which uses another table as a GenericRelation.
```
class Model(models.Model)
<...fields here>

generic = GenericRelation('throughModelname', ...)
```

On the view for this model, I declared:
```
filterset_fields = {
   "generic__id": ["exact", "in"]
}
```
but for some reason it doesn't recognize this field as a part of the model.

**Comment by carltongibson** [comment]
There's not really anything to go on here. 

See #830 for a discussion of generic filtering.
